UI/Model/CategoryController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Todo.Service.Model.Interface;
+
+namespace Todo.UI.Model
+{
+    /// <summary>
+    /// Category controller class.
+    /// </summary>
+    public sealed class CategoryController
```

Comment:
CategoryController and other files in Model aren't used and will be deleted.
We are using controllers directly now. But we can extract object relation functionality from VM to M in case of significant its size.